pkg/util/expansion/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
```

Comment:
Really? There's a ton of files with 2015 in their boilerplate, including the [boilerplate-checking script itself](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/hooks/boilerplate.py)